.github/workflows/run_release.yaml

```diff
@@ -43,6 +43,8 @@ jobs:
         !startsWith(github.event.head_commit.message, 'ci:')
       )
     runs-on: ubuntu-latest
+    env:
```

Comment:
maybe better move this to the top level (e.g. https://github.com/apify/crawlee/blob/master/.github/workflows/release.yml#L3)